CMakeLists.txt

```diff
@@ -203,7 +203,8 @@ if(CORENRN_ENABLE_NMODL)
     set(CORENRN_NMODL_BINARY ${CMAKE_BINARY_DIR}/bin/nmodl${CMAKE_EXECUTABLE_SUFFIX})
     set(CORENRN_NMODL_INCLUDE ${CMAKE_BINARY_DIR}/include)
     set(ENV{PYTHONPATH} "$ENV{PYTHONPATH}")
-    set(CORENRN_NMODL_PYTHONPATH $ENV{PYTHONPATH})
+    set(nmodl_PYTHONPATH "${CMAKE_BINARY_DIR}/lib/python")
```

Comment:
last time when I saw this was changed I thought it was on purpose and hence didn't ask question! :)